homeassistant/components/recorder/websocket_api.py

```diff
@@ -512,6 +532,7 @@ def valid_units(statistics_unit: str | None, adjustment_unit: str | None) -> boo
             vol.Required("source"): str,
             vol.Required("statistic_id"): str,
             vol.Required("unit_of_measurement"): vol.Any(str, None),
+            vol.Required("has_circular_mean"): bool,
```

Comment:
Should it be optional to avoid breaking changes?

Reply:
We don't usually make the WS API backwards compatible.
However, this WS command was added by a contributor, for them to use in their own project: https://github.com/home-assistant/core/pull/73937#issuecomment-1175254830

Some googling suggests it's used by that project, and maybe a couple of other cases. Considering the very limited use, I think it's OK to do a breaking change.

We can update the WS command in a separate PR, it doesn't have to happen in this PR.